packages/fiori/src/Bar.js

```diff
@@ -164,10 +166,21 @@ class Bar extends UI5Element {
 			root: {
 				"ui5-bar-root": true,
 				"ui5-bar-root-shrinked": this._shrinked,
+				"ui5-bar-root-rtl": this._isRTL,
 			},
 		};
 	}
 
+	get _isRTL() {
+		return this.effectiveDir === "rtl";
```

Comment:
you can just use effectiveDir in the template as value of dir attribute and then use the dir attribute in the CSS

If you do that, you can remove most of the code:
- "ui5-bar-root-rtl": this._isRTL,

- get _isRTL() {
		return this.effectiveDir === "rtl";
}

Reply:
done